warehouse/query-core/src/main/java/datawave/query/model/util/LoadModelFromXml.java

```diff
@@ -46,9 +46,7 @@ public static QueryModel loadModelFromXml(InputStream stream) throws Exception {
             switch (mapping.getDirection()) {
                 case FORWARD:
                     model.addTermToModel(mapping.getModelFieldName(), mapping.getFieldName());
-                    if (mapping.getIndexOnly()) {
-                        model.addUnevaluatedField(mapping.getFieldName());
-                    }
+                    model.addUnevaluatedField(mapping.getFieldName());
```

Comment:
This line should go away, and in addition, the concept of unevaluated (aka index-only) fields should be removed from the QueryModel class entirely. In addition (question for @ivakegg), I'm wondering if we need to write a utility to convert an old format model into a new one (e.g., to re-write the model in the metadata table)?

Reply:
My QueryModel.java file is read-only for some reason.  Am I barred from editing it for some reason?  I could change the perms and edit it but I just want to see that I am not barred for some reason.

Reply:
@brianloss A conversion utility is not a bad thought.  We are trying to be backward compatible but it would be good to save the extra bits in the metadata table.  @jzgithub1 How about create a separate ticket for creating this utility please.

Reply:
Will create a ticket to write a utility to convert an old format model into a new one (e.g., to re-write the model in the metadata table).  

I checked for backward compatibility in the pull request by doing an insert model with the indexOnly attribute in the xml that I posted.  The insert still worked.